analysis/wwz/get_wwz_tables.py

```diff
@@ -0,0 +1,158 @@
+#!/bin/env python
+
+import ewkcoffea.modules.sample_groupings as sg
+import ewkcoffea.modules.keynote_tools as kt
+import ewkcoffea.modules.yield_tools as yt
+
+import argparse
+import pickle
+import gzip
+
+# Get the yields in the SR
+def get_yields(histos_dict,sample_dict,raw_counts=False,quiet=True,blind=True,systematic_name="nominal"):
```

Comment:
What is the difference between this get_yields and the other two that already exist elsewhere? Is it possible to just use one of those instead of having another new one?

Reply:
I think this is not used and therefore should be deleted.